harness/tests/determined/common/experimental/test_experiment.py

```diff
@@ -229,3 +229,45 @@ def test_download_code_writes_output_to_file(
     with open(output_file, "rb") as f:
         file_content = f.read()
         assert file_content == b"b64TgzResponse"
+
+
+@mock.patch("determined.common.api.bindings.get_GetExperimentCheckpoints")
+@pytest.mark.parametrize(
```

Comment:
I don't tend to like these kinds of tests. If "am I passing through the right arguments" is tricky, then maybe something else has gone wrong. Already written, it doesn't add much overhead -- I guess the test will have to be updated/deleted whenever a parameter changes in either front end or back end -- so it's ok to leave in if you want to. It's just not the kind of thing I like writing tests for.

If there's some complicated calculation one of the parameters undergoes, nice -- I like testing that, specifically. Or if there's tricky edge casing re: parameters and how bindings are called, that's another good thing to test.

I don't know... maybe this is better as a conversation than as a comment. Either here or slack or voice, could you explain how you see it?

Reply:
> was testing that bindings receives the correct type, either enum or attr (str) from the caller.

That makes sense to me. What do you think about limiting the test's scope to exactly that? And then once that's done, maybe it makes sense to split it into two (also, maybe not)?

Reply:
Looks good! I like these new tests.

Reminder here to do the same for `test_trial.py` before merging.